frontend/src/metabase/lib/expressions/process.js

```diff
@@ -31,24 +28,9 @@ export function processSource(options) {
     }
   }
 
-  // SUGGEST
-  if (targetOffset != null) {
-    try {
-      ({ suggestions = [], helpText } = suggest({
```

Comment:
This entire block is gone since `suggest()` is invoked directly by the expression editor upon input change and cursor movement.